FirebaseMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 10.4.0
+- [fixed] Fixed a crash for strongSelf dereference (#10707).
+
```

Comment:
I had added a #nochangelog tag in the PR's description since it was more of an internal code flow bug fix. 

This is fine with me - maybe we should change the version to # 10.5.0

Reply:
Make sense, will revert the internal change.

Reply:
Sounds good - we should change the version number though since its part of 10.5.0.

Reply:
Oops, updated!

Reply:
+1 to changelogs for externally reported bugs and +1 to updating to 10.5.0